docs/enterprise/sso.md

```diff
@@ -49,119 +49,108 @@ openssl req -x509 -newkey rsa:2048 -keyout myservice.key -out myservice.cert -da
 
 lakeFS Server Configuration (Update in helm's `values.yaml` file):
 ```yaml
-lakefsConfig: |
-  # Important: make sure to include the rest of your lakeFS Configuration here!
-
-  auth:
-    cookie_auth_verification:
-      auth_source: saml
-      friendly_name_claim_name: displayName
-      external_user_id_claim_name: samName
-      default_initial_groups:
-        - "Developers"
-    logout_redirect_url: "https://lakefs.company.com/logout-saml"
-    encrypt:
-      secret_key: shared-secrey-key
-    ui_config:
-      login_url: "https://lakefs.company.com/sso/login-saml"
-      logout_url: "https://lakefs.company.com/sso/logout-saml"
-      login_cookie_names:
-        - internal_auth_session
-        - saml_auth_session
-      rbac: external
+auth:
+  cookie_auth_verification:
+    auth_source: saml
+    friendly_name_claim_name: displayName
+    external_user_id_claim_name: samName
+    default_initial_groups:
+      - "Developers"
+  logout_redirect_url: "https://lakefs.company.com/logout-saml"
+  encrypt:
+    secret_key: shared-secrey-key
+  ui_config:
+    login_url: "https://lakefs.company.com/sso/login-saml"
+    logout_url: "https://lakefs.company.com/sso/logout-saml"
+    login_cookie_names:
+      - internal_auth_session
+      - saml_auth_session
 ```
 
 Fluffy Configuration (Update in helm's `values.yaml` file):
 ```yaml
-fluffyConfig: &fluffyConfig |
-  logging:
-    format: "json"
-    level: "INFO"
-    audit_log_level: "INFO"
-    output: "="
-  installation:
-    fixed_id: fluffy-authenticator
-  auth:  
-    # better set from secret FLUFFY_AUTH_ENCRYPT_SECRET_KEY must be equal to what is used in lakeFS for auth_encrypt_secret_key
-    encrypt:
-      secret_key: shared-secrey-key    
-    logout_redirect_url: https://lakefs.company.com
-    post_login_redirect_url: https://lakefs.company.com
-    saml:
-      enabled: true 
-      sp_root_url: https://lakefs.company.com
-      sp_x509_key_path: '/etc/saml_certs/rsa_saml_private.cert'
-      sp_x509_cert_path: '/etc/saml_certs/rsa_saml_public.pem'
-      sp_sign_request: true
-      sp_signature_method: "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"
-      idp_metadata_url: "https://adfs-auth.company.com/federationmetadata/2007-06/federationmetadata.xml"
-      # idp_authn_name_id_format: "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified"
-      external_user_id_claim_name: samName
-      # idp_metadata_file_path: 
-      # idp_skip_verify_tls_cert: true
+logging:
+  format: "json"
+  level: "INFO"
+  audit_log_level: "INFO"
+  output: "="
+auth:  
+  encrypt:
+    # in Helm set via kind Secret
```

Comment:
Can we move this one as a note outside the config example?
I assume the yaml will be used to copy/paste and this comment is informative for the helm users.